pyproject.toml

```diff
@@ -2,6 +2,17 @@
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "ImageDePHI"
+version = "0.1.0"
+requires-python = ">=3.10"
```

Comment:
This was implicitly the Python version required, but it ought to be explicitly checked at install time.